test/onnx/test_operators.py

```diff
@@ -873,7 +873,7 @@ def test_std(self):
 
     def test_cumsum(self):
         x = torch.randn(2, 3, 4, requires_grad=True)
-        self.assertONNX(lambda x: torch.cumsum(x, dim=1), x, opset_version=11)
+        self.assertONNX(lambda *args: torch.cumsum(*args, dim=1), x, opset_version=11)
```

Comment:
This change fixes the failing test mentioned in https://github.com/pytorch/pytorch/pull/136224#issuecomment-2361912953

Apparently moving `torch.cumsum` to `torch/functional.py` caused the generated graph to pick up the name of the lambda arg "%x". I found that if I change the lambda arg to `my_arg`, for instance, the graph arg would be changed to "%my_arg" to match it.

So I figured I'd try using an args list instead--that way the generated graph isn't given a name for the arg, so it has to just use "%0"

Reply:
Feel free to do whatever that makes it work. I have a open PR to remove this test file completely but it is not in yet.